kahuna/public/js/components/gr-panel/gr-panel.js

```diff
@@ -53,15 +58,35 @@ grPanel.controller('GrPanel', [
             });
         };
 
+        ctrl.refreshUsageRights = () => {
+            selection.update();
+            ctrl.selectedCosts = selection.getCost();
+            ctrl.selectedUsageRights = selection.getUsageRights();
+        };
+
         $scope.$watch(() => selection.getMetadata(), onValChange(newMetadata => {
             ctrl.rawMetadata = newMetadata;
             ctrl.metadata = selection.getDisplayMetadata();
+            ctrl.images = Array.from(ctrl.selectedImages);
+
+            $rootScope.$broadcast('usage-rights:update-images', ctrl.images);
```

Comment:
Is `$broadcast` an acceptable thing to do here? Don't want to end up with event spaghetti.